swarm/content.md

```diff
@@ -0,0 +1,107 @@
+principle. It ships with a simple scheduling backend out of the box, and as initial
+development settles, an API will develop to enable pluggable backends. The goal is
+to provide a smooth out-of-box experience for simple use cases, and allow swapping
+in more powerful backends, like `Mesos`, for large scale production deployments.
+
+## Installation
+
+###1 - Download and install the current source code.
+Ensure you have golang and git client installed (e.g. `apt-get install golang git` on Ubuntu).
+You may need to set `$GOPATH`, e.g `mkdir ~/gocode; export GOPATH=~/gocode`.
+
+The install `swarm` binary to your `$GOPATH` directory.
+
+```sh
+go get -u github.com/docker/swarm
+```
+
+###2 - Nodes setup
+The only requirement for Swarm nodes is to run a regular Docker daemon (version
+`1.4.0` and later).
+
+In order for Swarm to be able to communicate with its nodes, they must bind on a
+network interface. This can be achieved by starting Docker with the `-H` flag
+(e.g. `-H tcp://0.0.0.0:2375`).
+
+# Example usage
+
+```bash
+# create a cluster
+$ swarm create
+6856663cdefdec325839a4b7e1de38e8 # <- this is your unique <cluster_id>
+
+# on each of your nodes, start the swarm agent
+#  <node_ip> doesn't have to be public (eg. 192.168.0.X),
+#  as long as the swarm manager can access it.
+$ swarm join --addr=<node_ip:2375> token://<cluster_id>
+
+# start the manager on any machine or your laptop
+$ swarm manage -H tcp://<swarm_ip:swarm_port> token://<cluster_id>
+
+# use the regular docker cli
+$ docker -H tcp://<swarm_ip:swarm_port> info
+$ docker -H tcp://<swarm_ip:swarm_port> run ...
+$ docker -H tcp://<swarm_ip:swarm_port> ps
+$ docker -H tcp://<swarm_ip:swarm_port> logs ...
+...
+
+# list nodes in your cluster
+$ swarm list token://<cluster_id>
+<node_ip:2375>
+```
+
+See [here](discovery) for more information about
+other discovery services.
+
+## Advanced Scheduling
+
+See [filters](scheduler/filter) and [strategies](scheduler/strategy) to learn
+more about advanced scheduling.
+
+## TLS
+
+Swarm supports TLS authentication between the CLI and Swarm but also between
+Swarm and the Docker nodes.
+
+In order to enable TLS, the same command line options as Docker can be specified:
+
+`swarm manage --tlsverify --tlscacert=<CACERT> --tlscert=<CERT> --tlskey=<KEY> [...]`
+
+Please refer to the [Docker documentation](https://docs.docker.com/articles/https/)
+for more information on how to set up TLS authentication on Docker and generating
+the certificates.
+
+Note that Swarm certificates must be generated with`extendedKeyUsage = clientAuth,serverAuth`.
+
+## Participating
+
+We welcome pull requests and patches; come say hi on IRC, #swarm on freenode.
+
+## Creators
+
+**Andrea Luzzardi**
+
+- <http://twitter.com/aluzzardi>
+- <http://github.com/aluzzardi>
+
+**Victor Vieux**
+
+- <http://twitter.com/vieux>
+- <http://github.com/vieux>
+
+## Copyright and license
```

Comment:
Thanks - I quickly scanned through other `content.md` and assumed it was the verbatim `README.md` so that's what I did (I just removed the travis status and logo since it wouldn't work on the hub).

I'll take your comments into account during the next update.